protocols/gossipsub/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-gossipsub"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Gossipsub protocol for libp2p"
-version = "0.48.0"
+version = "0.48.1"
```

Comment:
Version bump isn't needed, `libp2p-gossipsub-0.48.0` isn't released yet.

Reply:
oh! my bad, how do I keep track of the versions? I had made a similar error on a different PR

Reply:
Don't know tbh. I just manually check right now on crates.io what the latest version is. 
We used to have an `[unreleased]` suffix in the changelog if a version isn't released yet, but it was dropped at some point. I don't know why, IMO it was quite useful. @jxs do you remember the reason?

Reply:
yeah that's what I do as well, check `crates.io`. The reason it was removed was because without it we can just released patch versions without having to create PR's to update the `CHANGELOG.md` file, see [here](https://github.com/libp2p/rust-libp2p/pull/4894) for more info